### ISSUE TITLE
Docker environment for development - gas oracle stuck

### ISSUE THREAD

**Issue by henry-hz** [ethereum-optimism/optimism#2123]
Dear Optimism team,
I am trying to run the docker-compose-nobuild.yml file, and just noticed that the gas oracle is not up, with this output below. I tried to wait some minutes and restart the oracle, but still getting the same error.

Note the contradition:  "unable to connect to layer one .....    l2geth [means layer2]"

```
curl: (7) Failed to connect to l2geth port 8545 after 0 ms: Connection refused
ERROR[02-03|09:01:24.159] Unable to connect to layer two           addr=http://127.0.0.1:9545
ERROR[02-03|09:01:24.160] Unable to connect to layer one           addr=http://l2geth:8545
CRIT [02-03|09:01:24.161] application failed                       message="Post \"http://127.0.0.1:9545\": dial tcp 127.0.0.1:9545: connect: connection refused"
```

**Comment by kungmink** [comment]
{"signType":"ethSignTx","blockchain":"ETH","action":"coldWalletSign","sign":{"data":"0xb1a1a882000000000000000000000000000000000000000000000000000000000013d62000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000000","from":"0x87def097c417e1034399cc5c0f04d9a5022989e2","gas":"0x2542c","gasPrice":"0x2a2470d200","to":"0x99c9fc46f92e8a1c0dec1b1747d010903e884be1","value":"0x243d3d6bffba3185","nonce":"0xcb","chainId":"1"}}

**Comment by tynes** [comment]
The config for the `gas-oracle` has been changed but not updated in the `docker-compose` setup

**Comment by smartcontracts** [comment]
Sorry this took so long. I just fixed this within #2363. It hasn't been released yet (should be released in the next few days) but you can build locally from the `develop` branch to get the latest updates.
